INSTALLING.md

```diff
@@ -2,199 +2,178 @@
 
-# If you would like an out-of-the-box virtual environment to use,
-./create_venv.sh
-source python_env/bin/activate
-```
+- For more information, visit Tenstorrent's [TT-SMI GitHub repository](https://github.com/tenstorrent/tt-smi).
 
-where `ARCH_NAME` is one of `grayskull`, `wormhole_b0`, or `blackhole`,
-depending on your Tenstorrent card type.
+#### (Optional) Multi-Card Configuration (TT-Topology)
+- For TT-Loudbox or TT-QuietBox systems, visit Tenstorrent's [TT-Topology README](https://github.com/tenstorrent/tt-topology/blob/main/README.md).
 
-> [!NOTE]
->
-> Note about Python environments: You do not have to use `create_venv.sh`. If you
-> are less familiar with Python and its various environment tools, just use
-> `create_venv.sh` as shown above and the pre-built environment. If you choose
-> to install in your custom environment, please note that you may run into
-> compatibility issues between dependencies. It is up to the user to ensure
-> that all the packages in their environment are compatible with each other.
->
-> If you do choose to manage your own environment, please note that you must
-> use Pip 20.1.1 or lower to install this project. This is the highest version
-> of Pip that supports editable installs in the way that we use it.
+---
 
-#### Option 2: From wheel
+## TT-Metalium Installation:
 
-Download the latest wheel from our
-[releases](https://github.com/tenstorrent/tt-metal/releases/latest) page for
-the particular Tenstorrent card architecture that you have installed on your
-system. (ie. Grayskull, Wormhole, etc)
+> [!IMPORTANT]
+>
+> If you are using a release version of this software, check installation insttructions packaged with it.
 
-Install the wheel using your Python environment manager of choice. For example,
-to install with `pip`:
+### Step 1. Clone the Repository:
 
 ```sh
-pip install <wheel_file.whl>
+git clone https://github.com/tenstorrent/tt-metal.git --recurse-submodules
+cd tt-metal
+git submodule foreach 'git lfs fetch --all && git lfs pull'
 ```
 
-1. (For models users only) Set up environment for models
-
-If you are going to try our pre-built models in `models/`, then you must execute
-the following to:
-
-- install their required dependencies
-- set appropriate environment variables
-- set the CPU performance governor to ensure high performance on the host
-
+### Step 2. Setup environment variables and invoke our build scripts:
+
+- Note: Some advanced build configurations like unity builds require `CMake 3.20`.
+  To install `CMake 3.20` run:
+    ```sh
+    sudo apt remove cmake -y
+    pip3 install cmake --upgrade
+    hash -r
+    cmake --version
+    ```
+- For Grayskull:
 ```sh
+export ARCH_NAME=grayskull
+export TT_METAL_HOME=$(pwd)
 export PYTHONPATH=$(pwd)
-pip install -r tt_metal/python_env/requirements-dev.txt
-sudo apt-get install cpufrequtils
-sudo cpupower frequency-set -g performance
+./build_metal.sh
 ```
 
-#### Option 3: From Docker Release Image
```

Comment:
Is there a reason this section was deleted?

Reply:
I know Bryan and Jasmina were looking for this simplified option.

Reply:
It shouldn't have been. We were seeking clarity for the multiple installing options, we'll reintegrate it asap.

Reply:
I can explain some of the advantages for this with Docker we do not need the following steps as they are already included.
`Step 2. System-level dependencies`
`Step 4. Install and start using TT-NN and TT-Metalium!`

Reply:
So it might require some re-work of the installation instructions since it does not match the mold.